modules/common-workflows/pages/workflow-sp-upgrade-webui.adoc

```diff
@@ -18,22 +18,22 @@ They are used for illustration purposes only.
 [role=procedure]
 
 
-. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)"
+. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP5 x86_64 (BETA)"
```

Comment:
Why do we have (BETA) here? Please remove the BETA tag

Reply:
Ok looking again at the docs, I see there are screenshot attached to it so there will be mismatch. I would recommend in this case, don't change whole text and screenshots with every SP. Just add a line at the top that this workflow is for migrating from this version to this and user should take care of selecting the corresponding the channels if migrating from/to to a different version than what is stated here in this workflow.